packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -4897,13 +4898,16 @@ export class ProjectService extends BaseService {
         // it is possible that the user `abilities` are not uptodate
         // Before we check permissions on scheduleCompileProject
         // Permissions will be checked again with the uptodate user on scheduler
-        await this.scheduleCompileProject(
+        const { jobUuid } = await this.scheduleCompileProject(
```

Comment:
Main change is exposing this job uuid. The rest is mostly updating types and setting `setActiveJobId` in frontend.